app/lib/request.rb

```diff
@@ -261,7 +261,6 @@ def open(host, *args)
         outer_e = nil
         port    = args.first
 
-        addresses = []
```

Comment:
The autocorrect changed this to just be `[]` alone on the line, which is odd?

I do think the assignment is useless though ... will be  assigned via one of the below paths.